molecule/default/Dockerfile.j2

```
@@ -0,0 +1,4 @@
+FROM {{ item.image }}
+
+# Install dependencies
+RUN dnf upgrade -y && dnf install -y sudo python39
```

Don't care, the other version will work just fine, but you can have this shorter:

```suggestion
RUN dnf install --refresh -y sudo python39
```